builtin/settingtypes.txt

```diff
@@ -2056,14 +2056,14 @@ block_send_optimize_distance (Block send optimize distance) int 4 2 32767
 
 #    If enabled the server will perform map block occlusion culling based on
 #    on the eye position of the player. This can reduce the number of blocks
-#    sent to the client 50-80%. The client will not longer receive most invisible
+#    sent to the client 50-80%. The client will no longer receive most invisible
 #    so that the utility of noclip mode is reduced.
```

Comment:
Maybe I'm stupid, but what is this sentence even supposed to say?
Should it perhaps be:
```suggestion
#    sent to the client 50-80%. The client will no longer receive most invisible
#    blocks, so that the utility of noclip mode is reduced.
```